src/main/content/kabanero-install.html

```diff
@@ -37,7 +47,7 @@ <h3>Prerequisites</h3>
             <hr>
             <h3>Try the operator-based Kabanero Foundation Install</h3>
             <p>
-                Follow the <a href="/docs/ref/general/installing-kabanero-foundation.html">instructions to install Kabanero Foundation.</a>
+                Follow the <a href="/docs/{{docs-version[0]}}/ref/general/installing-kabanero-foundation.html">instructions to install Kabanero Foundation.</a>
```

Comment:
I wonder if somewhere else we can say "if there is no version in the URL then just redirect to latest version of the doc"

Sounds like it would be possible in `selectDocInToc` in docs.js. so `/docs/ref/general/installing-kabanero-foundation.html` would like to the latest doc of that.

This would stop the duplication of this liquid code.

This shouldn't be a problem because the website should always be linking to latest stuff anyways, and if it really needs to it can specify a version in the URL